### PR TITLE
Add ability to configure color of linked items based on regex

### PR DESCRIPTION
**Pull request by JasperCraeghs**
Regex solution for issue #34

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -345,6 +345,16 @@
 
 traceability_json_export_path = '_build/exported_items.json'
 
+# {regex: (default, :hover+:active, :visited)}
+# css-file will be built in the same order, which means that the bottom regex has highest priority
```
Comment (Letme) [inline doc/conf.py]: Can we add this into the docs as well?

Comment (JasperCraeghs) [reply doc/conf.py]: - After investigating more, I found out that insertion-order preservation nature of dict objects is new in Python 3.7. To keep the plugin's behaviour the same across all supported Python versions, we decided to use an `OrderedDict` for deterministic order in py<3.7 as well efdc9723860bb00707143d37b71be4d98a0fb6c2.
- Added new feature to readme b0b452638184ebdfd16ecb9b6ab51d49ade7f845.

File: README.rst [hunk 2]
```diff
@@ -342,6 +342,28 @@ Example of no callback per item:
 
     traceability_callback_per_item = None
 
+.. _traceability_config_link_colors:
+
+Custom colors for linked items
+==============================
+
+The plugin allows customization of the colors of traceable items in order to easily recognize the type of item which is
+linked to. A dictionary in the configuration file defines the regexm, which is used to match item IDs, as key and a
```
Comment (Letme) [inline README.rst]: `regexm` = `regexp`

File: README.rst [hunk 3]
```diff
@@ -342,6 +342,28 @@ Example of no callback per item:
...
+tuple of 1-3 color defining strings as value. The first color is used for the default hyperlink state, the second color
+is used for the hover and active states, and the third color is used to override the default color of the visted state.
+Leaving a color empty results in the use of the default html style. The top regex has the highest priority. To supported
```
Comment (Letme) [inline README.rst]: `supported` = `support`

File: README.rst (hunk 2 above)
Comment (JasperCraeghs) [reply README.rst]: 36e500df9a37f0e56e6f88bd570a185c982d4bcf

File: README.rst (hunk 3 above)
Comment (JasperCraeghs) [reply README.rst]: 36e500df9a37f0e56e6f88bd570a185c982d4bcf

**Pull request merged**